server/index.js

```diff
@@ -14,42 +14,66 @@ const port = 9000;
 app.use(cors());
 app.use(bodyParser.json());
 app.use(express.static('client/dist'));
+app.use('/:id', express.static('client/dist'));
```

Comment:
I don't think you need /:id here. Static is for requests to the root url normally